.travis.yml

```diff
@@ -18,12 +18,15 @@ services:
 before_script:
   - R CMD INSTALL .
 cache: packages
+repos:
```

Comment:
do you want to try using `Additional_repositories` in `DESCRIPTION` for this? I'm not sure I see how this is being used as-is in the travis build as it's not listed as a dependency (though I do see it in the dockerfile)

Reply:
So this is only in there because it is requried by naomi and we're including naomi in the build. I think there isn't a way for naomi to automatically install that for us?

Reply:
If naomi has an additional repos listed in *its* DESCRIPTION I believe it will work ok

Reply:
I'll make a note to update that tomorrow - I was handling it through travis like I was here.